arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -735,18 +739,25 @@
           "type": "string"
         },
         "certificate": {
-          "description": "The cluster certificate settings, the new certificate should exist in Vmss or KeyVault,before you add it, it will override original value",
+          "description": "The cluster certificate settings, the new certificate should exist in VMSS or KeyVault,before you add it, it will override original value",
           "$ref": "#/definitions/CertificateDescription"
         },
         "clientCertificateThumbprints": {
-          "description": "The cluster client thumbprints, it will override existing collection",
+          "description": "The client thumbprint details ,it is used for client access for cluter operation, it will override existing collection",
```

Comment:
w certificate should exist in VMSS or KeyVault,before --> missing space after KeyVault